src/hotspot/share/runtime/os.cpp

```diff
@@ -22,6 +22,9 @@
  *
  */
 
+// Stopgap fix until FORBID_C_FUNCTION can work properly with LTO
+#define DISABLE_POISONING_STOPGAP
```

Comment:
So far as I can tell, this doesn't work.  I still get tons of `-Wattribute-warning`s when building
with LTO, because of similar problem from other files.